numprocs=48

Reading command line arguments using syntax: python file.py [simConfig=filepath] [netParams=filepath]
Loading file ../data/v8_batch4/v8_batch4_0_0_cfg.json ... 
Loading simConfig...
Importing netParams from ../data/v8_batch4/v8_batch4_netParams.py
  Number of cells on node 32: 1 
  Number of connections on node 32: 2 
  Number of synaptic contacts on node 32: 22 
  Number of stims on node 32: 11 
Recording 1 traces of 1 types on node 32
Plotting LFP ...
  There was an exception in plotLFP(): 
    module 'netpyne.sim' has no attribute 'allSimData' 
    (<class 'AttributeError'>, AttributeError("module 'netpyne.sim' has no attribute 'allSimData'"), <traceback object at 0x7ff0d9b11960>)
Plotting LFP ...
  There was an exception in plotLFP(): 
    module 'netpyne.sim' has no attribute 'allSimData' 
    (<class 'AttributeError'>, AttributeError("module 'netpyne.sim' has no attribute 'allSimData'"), <traceback object at 0x7ff0d9b11960>)

Reading command line arguments using syntax: python file.py [simConfig=filepath] [netParams=filepath]
Loading file ../data/v8_batch4/v8_batch4_0_0_cfg.json ... 
Loading simConfig...
Importing netParams from ../data/v8_batch4/v8_batch4_netParams.py
  Number of cells on node 16: 2 
  Number of connections on node 16: 12 
  Number of synaptic contacts on node 16: 58 
  Number of stims on node 16: 22 
Recording 2 traces of 1 types on node 16
Plotting LFP ...
  There was an exception in plotLFP(): 
    module 'netpyne.sim' has no attribute 'allSimData' 
    (<class 'AttributeError'>, AttributeError("module 'netpyne.sim' has no attribute 'allSimData'"), <traceback object at 0x7fdb061b0eb0>)
Plotting LFP ...
  There was an exception in plotLFP(): 
    module 'netpyne.sim' has no attribute 'allSimData' 
    (<class 'AttributeError'>, AttributeError("module 'netpyne.sim' has no attribute 'allSimData'"), <traceback object at 0x7fdb061b0eb0>)

Reading command line arguments using syntax: python file.py [simConfig=filepath] [netParams=filepath]
Loading file ../data/v8_batch4/v8_batch4_0_0_cfg.json ... 
Loading simConfig...
Importing netParams from ../data/v8_batch4/v8_batch4_netParams.py
  Number of cells on node 1: 2 
  Number of connections on node 1: 17 
  Number of synaptic contacts on node 1: 128 
  Number of stims on node 1: 22 
Recording 2 traces of 1 types on node 1
Plotting LFP ...
  There was an exception in plotLFP(): 
    module 'netpyne.sim' has no attribute 'allSimData' 
    (<class 'AttributeError'>, AttributeError("module 'netpyne.sim' has no attribute 'allSimData'"), <traceback object at 0x7f2c4a870f50>)
Plotting LFP ...
  There was an exception in plotLFP(): 
    module 'netpyne.sim' has no attribute 'allSimData' 
    (<class 'AttributeError'>, AttributeError("module 'netpyne.sim' has no attribute 'allSimData'"), <traceback object at 0x7f2c4a870f50>)

Reading command line arguments using syntax: python file.py [simConfig=filepath] [netParams=filepath]
Loading file ../data/v8_batch4/v8_batch4_0_0_cfg.json ... 
Loading simConfig...
Importing netParams from ../data/v8_batch4/v8_batch4_netParams.py
  Number of cells on node 8: 2 
  Number of connections on node 8: 9 
  Number of synaptic contacts on node 8: 65 
  Number of stims on node 8: 22 
Recording 2 traces of 1 types on node 8
Plotting LFP ...
  There was an exception in plotLFP(): 
    module 'netpyne.sim' has no attribute 'allSimData' 
    (<class 'AttributeError'>, AttributeError("module 'netpyne.sim' has no attribute 'allSimData'"), <traceback object at 0x7fc88d0c85f0>)
Plotting LFP ...
  There was an exception in plotLFP(): 
    module 'netpyne.sim' has no attribute 'allSimData' 
    (<class 'AttributeError'>, AttributeError("module 'netpyne.sim' has no attribute 'allSimData'"), <traceback object at 0x7fc88d0c85f0>)

Reading command line arguments using syntax: python file.py [simConfig=filepath] [netParams=filepath]
Loading file ../data/v8_batch4/v8_batch4_0_0_cfg.json ... 
Loading simConfig...
Importing netParams from ../data/v8_batch4/v8_batch4_netParams.py
  Number of cells on node 18: 1 
  Number of connections on node 18: 2 
  Number of synaptic contacts on node 18: 15 
  Number of stims on node 18: 11 
Recording 1 traces of 1 types on node 18
Plotting LFP ...
  There was an exception in plotLFP(): 
    module 'netpyne.sim' has no attribute 'allSimData' 
    (<class 'AttributeError'>, AttributeError("module 'netpyne.sim' has no attribute 'allSimData'"), <traceback object at 0x7f91d494c280>)
Plotting LFP ...
  There was an exception in plotLFP(): 
    module 'netpyne.sim' has no attribute 'allSimData' 
    (<class 'AttributeError'>, AttributeError("module 'netpyne.sim' has no attribute 'allSimData'"), <traceback object at 0x7f91d494c280>)

Reading command line arguments using syntax: python file.py [simConfig=filepath] [netParams=filepath]
Loading file ../data/v8_batch4/v8_batch4_0_0_cfg.json ... 
Loading simConfig...
Importing netParams from ../data/v8_batch4/v8_batch4_netParams.py
  Number of cells on node 12: 2 
  Number of connections on node 12: 5 
  Number of synaptic contacts on node 12: 40 
  Number of stims on node 12: 22 
Recording 2 traces of 1 types on node 12
Plotting LFP ...
  There was an exception in plotLFP(): 
    module 'netpyne.sim' has no attribute 'allSimData' 
    (<class 'AttributeError'>, AttributeError("module 'netpyne.sim' has no attribute 'allSimData'"), <traceback object at 0x7fe80877f500>)
Plotting LFP ...
  There was an exception in plotLFP(): 
    module 'netpyne.sim' has no attribute 'allSimData' 
    (<class 'AttributeError'>, AttributeError("module 'netpyne.sim' has no attribute 'allSimData'"), <traceback object at 0x7fe80877f500>)

Reading command line arguments using syntax: python file.py [simConfig=filepath] [netParams=filepath]
Loading file ../data/v8_batch4/v8_batch4_0_0_cfg.json ... 
Loading simConfig...
Importing netParams from ../data/v8_batch4/v8_batch4_netParams.py
  Number of cells on node 9: 2 
  Number of connections on node 9: 4 
  Number of synaptic contacts on node 9: 31 
  Number of stims on node 9: 22 
Recording 2 traces of 1 types on node 9
Plotting LFP ...
  There was an exception in plotLFP(): 
    module 'netpyne.sim' has no attribute 'allSimData' 
    (<class 'AttributeError'>, AttributeError("module 'netpyne.sim' has no attribute 'allSimData'"), <traceback object at 0x7ff28acf7b40>)
Plotting LFP ...
  There was an exception in plotLFP(): 
    module 'netpyne.sim' has no attribute 'allSimData' 
    (<class 'AttributeError'>, AttributeError("module 'netpyne.sim' has no attribute 'allSimData'"), <traceback object at 0x7ff28acf7b40>)

Reading command line arguments using syntax: python file.py [simConfig=filepath] [netParams=filepath]
Loading file ../data/v8_batch4/v8_batch4_0_0_cfg.json ... 
Loading simConfig...
Importing netParams from ../data/v8_batch4/v8_batch4_netParams.py
  Number of cells on node 33: 1 
  Number of connections on node 33: 0 
  Number of stims on node 33: 11 
Recording 1 traces of 1 types on node 33
Plotting LFP ...
  There was an exception in plotLFP(): 
    module 'netpyne.sim' has no attribute 'allSimData' 
    (<class 'AttributeError'>, AttributeError("module 'netpyne.sim' has no attribute 'allSimData'"), <traceback object at 0x7ff90a685aa0>)
Plotting LFP ...
  There was an exception in plotLFP(): 
    module 'netpyne.sim' has no attribute 'allSimData' 
    (<class 'AttributeError'>, AttributeError("module 'netpyne.sim' has no attribute 'allSimData'"), <traceback object at 0x7ff90a685aa0>)

Reading command line arguments using syntax: python file.py [simConfig=filepath] [netParams=filepath]
Loading file ../data/v8_batch4/v8_batch4_0_0_cfg.json ... 
Loading simConfig...
Importing netParams from ../data/v8_batch4/v8_batch4_netParams.py
  Number of cells on node 11: 2 
  Number of connections on node 11: 10 
  Number of synaptic contacts on node 11: 48 
  Number of stims on node 11: 22 
Recording 2 traces of 1 types on node 11
Plotting LFP ...
  There was an exception in plotLFP(): 
    module 'netpyne.sim' has no attribute 'allSimData' 
    (<class 'AttributeError'>, AttributeError("module 'netpyne.sim' has no attribute 'allSimData'"), <traceback object at 0x7fd5faa9dbe0>)
Plotting LFP ...
  There was an exception in plotLFP(): 
    module 'netpyne.sim' has no attribute 'allSimData' 
    (<class 'AttributeError'>, AttributeError("module 'netpyne.sim' has no attribute 'allSimData'"), <traceback object at 0x7fd5faa9dbe0>)

Reading command line arguments using syntax: python file.py [simConfig=filepath] [netParams=filepath]
Loading file ../data/v8_batch4/v8_batch4_0_0_cfg.json ... 
Loading simConfig...
Importing netParams from ../data/v8_batch4/v8_batch4_netParams.py
  Number of cells on node 10: 2 
  Number of connections on node 10: 11 
  Number of synaptic contacts on node 10: 89 
  Number of stims on node 10: 22 
Recording 2 traces of 1 types on node 10
Plotting LFP ...
  There was an exception in plotLFP(): 
    module 'netpyne.sim' has no attribute 'allSimData' 
    (<class 'AttributeError'>, AttributeError("module 'netpyne.sim' has no attribute 'allSimData'"), <traceback object at 0x7f82e3a5b0f0>)
Plotting LFP ...
  There was an exception in plotLFP(): 
    module 'netpyne.sim' has no attribute 'allSimData' 
    (<class 'AttributeError'>, AttributeError("module 'netpyne.sim' has no attribute 'allSimData'"), <traceback object at 0x7f82e3a5b0f0>)

Reading command line arguments using syntax: python file.py [simConfig=filepath] [netParams=filepath]
Loading file ../data/v8_batch4/v8_batch4_0_0_cfg.json ... 
Loading simConfig...
Importing netParams from ../data/v8_batch4/v8_batch4_netParams.py
  Number of cells on node 34: 1 
  Number of connections on node 34: 2 
  Number of synaptic contacts on node 34: 18 
  Number of stims on node 34: 11 
Recording 1 traces of 1 types on node 34
Plotting LFP ...
  There was an exception in plotLFP(): 
    module 'netpyne.sim' has no attribute 'allSimData' 
    (<class 'AttributeError'>, AttributeError("module 'netpyne.sim' has no attribute 'allSimData'"), <traceback object at 0x7efcbee37320>)
Plotting LFP ...
  There was an exception in plotLFP(): 
    module 'netpyne.sim' has no attribute 'allSimData' 
    (<class 'AttributeError'>, AttributeError("module 'netpyne.sim' has no attribute 'allSimData'"), <traceback object at 0x7efcbee37320>)

Reading command line arguments using syntax: python file.py [simConfig=filepath] [netParams=filepath]
Loading file ../data/v8_batch4/v8_batch4_0_0_cfg.json ... 
Loading simConfig...
Importing netParams from ../data/v8_batch4/v8_batch4_netParams.py
  Number of cells on node 36: 1 
  Number of connections on node 36: 3 
  Number of synaptic contacts on node 36: 27 
  Number of stims on node 36: 11 
Recording 1 traces of 1 types on node 36
Plotting LFP ...
  There was an exception in plotLFP(): 
    module 'netpyne.sim' has no attribute 'allSimData' 
    (<class 'AttributeError'>, AttributeError("module 'netpyne.sim' has no attribute 'allSimData'"), <traceback object at 0x7f43fa2fe230>)
Plotting LFP ...
  There was an exception in plotLFP(): 
    module 'netpyne.sim' has no attribute 'allSimData' 
    (<class 'AttributeError'>, AttributeError("module 'netpyne.sim' has no attribute 'allSimData'"), <traceback object at 0x7f43f9eaa230>)

Reading command line arguments using syntax: python file.py [simConfig=filepath] [netParams=filepath]
Loading file ../data/v8_batch4/v8_batch4_0_0_cfg.json ... 
Loading simConfig...
Importing netParams from ../data/v8_batch4/v8_batch4_netParams.py
  Number of cells on node 40: 1 
  Number of connections on node 40: 1 
  Number of synaptic contacts on node 40: 8 
  Number of stims on node 40: 11 
Recording 1 traces of 1 types on node 40
Plotting LFP ...
  There was an exception in plotLFP(): 
    module 'netpyne.sim' has no attribute 'allSimData' 
    (<class 'AttributeError'>, AttributeError("module 'netpyne.sim' has no attribute 'allSimData'"), <traceback object at 0x7f514c4aee60>)
Plotting LFP ...
  There was an exception in plotLFP(): 
    module 'netpyne.sim' has no attribute 'allSimData' 
    (<class 'AttributeError'>, AttributeError("module 'netpyne.sim' has no attribute 'allSimData'"), <traceback object at 0x7f514c4aee60>)

Reading command line arguments using syntax: python file.py [simConfig=filepath] [netParams=filepath]
Loading file ../data/v8_batch4/v8_batch4_0_0_cfg.json ... 
Loading simConfig...
Importing netParams from ../data/v8_batch4/v8_batch4_netParams.py
  Number of cells on node 38: 1 
  Number of connections on node 38: 3 
  Number of synaptic contacts on node 38: 26 
  Number of stims on node 38: 11 
Recording 1 traces of 1 types on node 38
Plotting LFP ...
  There was an exception in plotLFP(): 
    module 'netpyne.sim' has no attribute 'allSimData' 
    (<class 'AttributeError'>, AttributeError("module 'netpyne.sim' has no attribute 'allSimData'"), <traceback object at 0x7f3e97207780>)
Plotting LFP ...
  There was an exception in plotLFP(): 
    module 'netpyne.sim' has no attribute 'allSimData' 
    (<class 'AttributeError'>, AttributeError("module 'netpyne.sim' has no attribute 'allSimData'"), <traceback object at 0x7f3e97207780>)

Reading command line arguments using syntax: python file.py [simConfig=filepath] [netParams=filepath]
Loading file ../data/v8_batch4/v8_batch4_0_0_cfg.json ... 
Loading simConfig...
Importing netParams from ../data/v8_batch4/v8_batch4_netParams.py
  Number of cells on node 41: 1 
  Number of connections on node 41: 10 
  Number of synaptic contacts on node 41: 76 
  Number of stims on node 41: 11 
Recording 1 traces of 1 types on node 41
Plotting LFP ...
  There was an exception in plotLFP(): 
    module 'netpyne.sim' has no attribute 'allSimData' 
    (<class 'AttributeError'>, AttributeError("module 'netpyne.sim' has no attribute 'allSimData'"), <traceback object at 0x7fcc16b80370>)
Plotting LFP ...
  There was an exception in plotLFP(): 
    module 'netpyne.sim' has no attribute 'allSimData' 
    (<class 'AttributeError'>, AttributeError("module 'netpyne.sim' has no attribute 'allSimData'"), <traceback object at 0x7fcc16b80370>)

Reading command line arguments using syntax: python file.py [simConfig=filepath] [netParams=filepath]
Loading file ../data/v8_batch4/v8_batch4_0_0_cfg.json ... 
Loading simConfig...
Importing netParams from ../data/v8_batch4/v8_batch4_netParams.py
  Number of cells on node 2: 2 
  Number of connections on node 2: 11 
  Number of synaptic contacts on node 2: 84 
  Number of stims on node 2: 22 
Recording 2 traces of 1 types on node 2
Plotting LFP ...
  There was an exception in plotLFP(): 
    module 'netpyne.sim' has no attribute 'allSimData' 
    (<class 'AttributeError'>, AttributeError("module 'netpyne.sim' has no attribute 'allSimData'"), <traceback object at 0x7f221111cc30>)
Plotting LFP ...
  There was an exception in plotLFP(): 
    module 'netpyne.sim' has no attribute 'allSimData' 
    (<class 'AttributeError'>, AttributeError("module 'netpyne.sim' has no attribute 'allSimData'"), <traceback object at 0x7f221111cc30>)

Reading command line arguments using syntax: python file.py [simConfig=filepath] [netParams=filepath]
Loading file ../data/v8_batch4/v8_batch4_0_0_cfg.json ... 
Loading simConfig...
Importing netParams from ../data/v8_batch4/v8_batch4_netParams.py
  Number of cells on node 37: 1 
  Number of connections on node 37: 3 
  Number of synaptic contacts on node 37: 24 
  Number of stims on node 37: 11 
Recording 1 traces of 1 types on node 37
Plotting LFP ...
  There was an exception in plotLFP(): 
    module 'netpyne.sim' has no attribute 'allSimData' 
    (<class 'AttributeError'>, AttributeError("module 'netpyne.sim' has no attribute 'allSimData'"), <traceback object at 0x7fc5bf0a17d0>)
Plotting LFP ...
  There was an exception in plotLFP(): 
    module 'netpyne.sim' has no attribute 'allSimData' 
    (<class 'AttributeError'>, AttributeError("module 'netpyne.sim' has no attribute 'allSimData'"), <traceback object at 0x7fc5bf0a17d0>)

Reading command line arguments using syntax: python file.py [simConfig=filepath] [netParams=filepath]
Loading file ../data/v8_batch4/v8_batch4_0_0_cfg.json ... 
Loading simConfig...
Importing netParams from ../data/v8_batch4/v8_batch4_netParams.py
  Number of cells on node 13: 2 
  Number of connections on node 13: 10 
  Number of synaptic contacts on node 13: 71 
  Number of stims on node 13: 22 
Recording 2 traces of 1 types on node 13
Plotting LFP ...
  There was an exception in plotLFP(): 
    module 'netpyne.sim' has no attribute 'allSimData' 
    (<class 'AttributeError'>, AttributeError("module 'netpyne.sim' has no attribute 'allSimData'"), <traceback object at 0x7fc9ac133fa0>)
Plotting LFP ...
  There was an exception in plotLFP(): 
    module 'netpyne.sim' has no attribute 'allSimData' 
    (<class 'AttributeError'>, AttributeError("module 'netpyne.sim' has no attribute 'allSimData'"), <traceback object at 0x7fc9ac133fa0>)

Reading command line arguments using syntax: python file.py [simConfig=filepath] [netParams=filepath]
Loading file ../data/v8_batch4/v8_batch4_0_0_cfg.json ... 
Loading simConfig...
Importing netParams from ../data/v8_batch4/v8_batch4_netParams.py
  Number of cells on node 35: 1 
  Number of connections on node 35: 2 
  Number of synaptic contacts on node 35: 18 
  Number of stims on node 35: 11 
Recording 1 traces of 1 types on node 35
Plotting LFP ...
  There was an exception in plotLFP(): 
    module 'netpyne.sim' has no attribute 'allSimData' 
    (<class 'AttributeError'>, AttributeError("module 'netpyne.sim' has no attribute 'allSimData'"), <traceback object at 0x7fdd9a078320>)
Plotting LFP ...
  There was an exception in plotLFP(): 
    module 'netpyne.sim' has no attribute 'allSimData' 
    (<class 'AttributeError'>, AttributeError("module 'netpyne.sim' has no attribute 'allSimData'"), <traceback object at 0x7fdd9a078320>)

Reading command line arguments using syntax: python file.py [simConfig=filepath] [netParams=filepath]
Loading file ../data/v8_batch4/v8_batch4_0_0_cfg.json ... 
Loading simConfig...
Importing netParams from ../data/v8_batch4/v8_batch4_netParams.py
  Number of cells on node 5: 2 
  Number of connections on node 5: 8 
  Number of synaptic contacts on node 5: 42 
  Number of stims on node 5: 22 
Recording 2 traces of 1 types on node 5
Plotting LFP ...
  There was an exception in plotLFP(): 
    module 'netpyne.sim' has no attribute 'allSimData' 
    (<class 'AttributeError'>, AttributeError("module 'netpyne.sim' has no attribute 'allSimData'"), <traceback object at 0x7ff464611460>)
Plotting LFP ...
  There was an exception in plotLFP(): 
    module 'netpyne.sim' has no attribute 'allSimData' 
    (<class 'AttributeError'>, AttributeError("module 'netpyne.sim' has no attribute 'allSimData'"), <traceback object at 0x7ff464611460>)

Reading command line arguments using syntax: python file.py [simConfig=filepath] [netParams=filepath]
Loading file ../data/v8_batch4/v8_batch4_0_0_cfg.json ... 
Loading simConfig...
Importing netParams from ../data/v8_batch4/v8_batch4_netParams.py
  Number of cells on node 4: 2 
  Number of connections on node 4: 10 
  Number of synaptic contacts on node 4: 74 
  Number of stims on node 4: 22 
Recording 2 traces of 1 types on node 4
Plotting LFP ...
  There was an exception in plotLFP(): 
    module 'netpyne.sim' has no attribute 'allSimData' 
    (<class 'AttributeError'>, AttributeError("module 'netpyne.sim' has no attribute 'allSimData'"), <traceback object at 0x7fef2f710be0>)
Plotting LFP ...
  There was an exception in plotLFP(): 
    module 'netpyne.sim' has no attribute 'allSimData' 
    (<class 'AttributeError'>, AttributeError("module 'netpyne.sim' has no attribute 'allSimData'"), <traceback object at 0x7fef2a6fa320>)

Reading command line arguments using syntax: python file.py [simConfig=filepath] [netParams=filepath]
Loading file ../data/v8_batch4/v8_batch4_0_0_cfg.json ... 
Loading simConfig...
Importing netParams from ../data/v8_batch4/v8_batch4_netParams.py
  Number of cells on node 24: 1 
  Number of connections on node 24: 5 
  Number of synaptic contacts on node 24: 49 
  Number of stims on node 24: 11 
Recording 1 traces of 1 types on node 24
Plotting LFP ...
  There was an exception in plotLFP(): 
    module 'netpyne.sim' has no attribute 'allSimData' 
    (<class 'AttributeError'>, AttributeError("module 'netpyne.sim' has no attribute 'allSimData'"), <traceback object at 0x7fa841a4dbe0>)
Plotting LFP ...
  There was an exception in plotLFP(): 
    module 'netpyne.sim' has no attribute 'allSimData' 
    (<class 'AttributeError'>, AttributeError("module 'netpyne.sim' has no attribute 'allSimData'"), <traceback object at 0x7fa83cb0dfa0>)

Reading command line arguments using syntax: python file.py [simConfig=filepath] [netParams=filepath]
Loading file ../data/v8_batch4/v8_batch4_0_0_cfg.json ... 
Loading simConfig...
Importing netParams from ../data/v8_batch4/v8_batch4_netParams.py
  Number of cells on node 20: 1 
  Number of connections on node 20: 2 
  Number of synaptic contacts on node 20: 8 
  Number of stims on node 20: 11 
Recording 1 traces of 1 types on node 20
Plotting LFP ...
  There was an exception in plotLFP(): 
    module 'netpyne.sim' has no attribute 'allSimData' 
    (<class 'AttributeError'>, AttributeError("module 'netpyne.sim' has no attribute 'allSimData'"), <traceback object at 0x7f32a0d93af0>)
Plotting LFP ...
  There was an exception in plotLFP(): 
    module 'netpyne.sim' has no attribute 'allSimData' 
    (<class 'AttributeError'>, AttributeError("module 'netpyne.sim' has no attribute 'allSimData'"), <traceback object at 0x7f32a0e6bdc0>)

Reading command line arguments using syntax: python file.py [simConfig=filepath] [netParams=filepath]
Loading file ../data/v8_batch4/v8_batch4_0_0_cfg.json ... 
Loading simConfig...
Importing netParams from ../data/v8_batch4/v8_batch4_netParams.py
  Number of cells on node 22: 1 
  Number of connections on node 22: 5 
  Number of synaptic contacts on node 22: 52 
  Number of stims on node 22: 11 
Recording 1 traces of 1 types on node 22
Plotting LFP ...
  There was an exception in plotLFP(): 
    module 'netpyne.sim' has no attribute 'allSimData' 
    (<class 'AttributeError'>, AttributeError("module 'netpyne.sim' has no attribute 'allSimData'"), <traceback object at 0x7fe4aef16be0>)
Plotting LFP ...
  There was an exception in plotLFP(): 
    module 'netpyne.sim' has no attribute 'allSimData' 
    (<class 'AttributeError'>, AttributeError("module 'netpyne.sim' has no attribute 'allSimData'"), <traceback object at 0x7fe4aef16be0>)

Reading command line arguments using syntax: python file.py [simConfig=filepath] [netParams=filepath]
Loading file ../data/v8_batch4/v8_batch4_0_0_cfg.json ... 
Loading simConfig...
Importing netParams from ../data/v8_batch4/v8_batch4_netParams.py
  Number of cells on node 21: 1 
  Number of connections on node 21: 4 
  Number of synaptic contacts on node 21: 15 
  Number of stims on node 21: 11 
Recording 1 traces of 1 types on node 21
Plotting LFP ...
  There was an exception in plotLFP(): 
    module 'netpyne.sim' has no attribute 'allSimData' 
    (<class 'AttributeError'>, AttributeError("module 'netpyne.sim' has no attribute 'allSimData'"), <traceback object at 0x7f2e70f3eb40>)
Plotting LFP ...
  There was an exception in plotLFP(): 
    module 'netpyne.sim' has no attribute 'allSimData' 
    (<class 'AttributeError'>, AttributeError("module 'netpyne.sim' has no attribute 'allSimData'"), <traceback object at 0x7f2e71017e10>)

Reading command line arguments using syntax: python file.py [simConfig=filepath] [netParams=filepath]
Loading file ../data/v8_batch4/v8_batch4_0_0_cfg.json ... 
Loading simConfig...
Importing netParams from ../data/v8_batch4/v8_batch4_netParams.py
  Number of cells on node 23: 1 
  Number of connections on node 23: 1 
  Number of synaptic contacts on node 23: 4 
  Number of stims on node 23: 11 
Recording 1 traces of 1 types on node 23
Plotting LFP ...
  There was an exception in plotLFP(): 
    module 'netpyne.sim' has no attribute 'allSimData' 
    (<class 'AttributeError'>, AttributeError("module 'netpyne.sim' has no attribute 'allSimData'"), <traceback object at 0x7f333b2895a0>)
Plotting LFP ...
  There was an exception in plotLFP(): 
    module 'netpyne.sim' has no attribute 'allSimData' 
    (<class 'AttributeError'>, AttributeError("module 'netpyne.sim' has no attribute 'allSimData'"), <traceback object at 0x7f333b2895a0>)

Reading command line arguments using syntax: python file.py [simConfig=filepath] [netParams=filepath]
Loading file ../data/v8_batch4/v8_batch4_0_0_cfg.json ... 
Loading simConfig...
Importing netParams from ../data/v8_batch4/v8_batch4_netParams.py
  Number of cells on node 44: 1 
  Number of connections on node 44: 10 
  Number of synaptic contacts on node 44: 74 
  Number of stims on node 44: 11 
Recording 1 traces of 1 types on node 44
Plotting LFP ...
  There was an exception in plotLFP(): 
    module 'netpyne.sim' has no attribute 'allSimData' 
    (<class 'AttributeError'>, AttributeError("module 'netpyne.sim' has no attribute 'allSimData'"), <traceback object at 0x7f1506a86be0>)
Plotting LFP ...
  There was an exception in plotLFP(): 
    module 'netpyne.sim' has no attribute 'allSimData' 
    (<class 'AttributeError'>, AttributeError("module 'netpyne.sim' has no attribute 'allSimData'"), <traceback object at 0x7f1506a86be0>)

Reading command line arguments using syntax: python file.py [simConfig=filepath] [netParams=filepath]
Loading file ../data/v8_batch4/v8_batch4_0_0_cfg.json ... 
Loading simConfig...
Importing netParams from ../data/v8_batch4/v8_batch4_netParams.py
  Number of cells on node 45: 1 
  Number of connections on node 45: 5 
  Number of synaptic contacts on node 45: 41 
  Number of stims on node 45: 11 
Recording 1 traces of 1 types on node 45
Plotting LFP ...
  There was an exception in plotLFP(): 
    module 'netpyne.sim' has no attribute 'allSimData' 
    (<class 'AttributeError'>, AttributeError("module 'netpyne.sim' has no attribute 'allSimData'"), <traceback object at 0x7f63e9474c80>)
Plotting LFP ...
  There was an exception in plotLFP(): 
    module 'netpyne.sim' has no attribute 'allSimData' 
    (<class 'AttributeError'>, AttributeError("module 'netpyne.sim' has no attribute 'allSimData'"), <traceback object at 0x7f63e9474c80>)

Reading command line arguments using syntax: python file.py [simConfig=filepath] [netParams=filepath]
Loading file ../data/v8_batch4/v8_batch4_0_0_cfg.json ... 
Loading simConfig...
Importing netParams from ../data/v8_batch4/v8_batch4_netParams.py
  Number of cells on node 29: 1 
  Number of connections on node 29: 4 
  Number of synaptic contacts on node 29: 30 
  Number of stims on node 29: 11 
Recording 1 traces of 1 types on node 29
Plotting LFP ...
  There was an exception in plotLFP(): 
    module 'netpyne.sim' has no attribute 'allSimData' 
    (<class 'AttributeError'>, AttributeError("module 'netpyne.sim' has no attribute 'allSimData'"), <traceback object at 0x7f5c78bc70f0>)
Plotting LFP ...
  There was an exception in plotLFP(): 
    module 'netpyne.sim' has no attribute 'allSimData' 
    (<class 'AttributeError'>, AttributeError("module 'netpyne.sim' has no attribute 'allSimData'"), <traceback object at 0x7f5c78bc70f0>)

Reading command line arguments using syntax: python file.py [simConfig=filepath] [netParams=filepath]
Loading file ../data/v8_batch4/v8_batch4_0_0_cfg.json ... 
Loading simConfig...
Importing netParams from ../data/v8_batch4/v8_batch4_netParams.py
  Number of cells on node 28: 1 
  Number of connections on node 28: 1 
  Number of synaptic contacts on node 28: 4 
  Number of stims on node 28: 11 
Recording 1 traces of 1 types on node 28
Plotting LFP ...
  There was an exception in plotLFP(): 
    module 'netpyne.sim' has no attribute 'allSimData' 
    (<class 'AttributeError'>, AttributeError("module 'netpyne.sim' has no attribute 'allSimData'"), <traceback object at 0x7f7a49016d70>)
Plotting LFP ...
  There was an exception in plotLFP(): 
    module 'netpyne.sim' has no attribute 'allSimData' 
    (<class 'AttributeError'>, AttributeError("module 'netpyne.sim' has no attribute 'allSimData'"), <traceback object at 0x7f7a49016d70>)

Reading command line arguments using syntax: python file.py [simConfig=filepath] [netParams=filepath]
Loading file ../data/v8_batch4/v8_batch4_0_0_cfg.json ... 
Loading simConfig...
Importing netParams from ../data/v8_batch4/v8_batch4_netParams.py
  Number of cells on node 39: 1 
  Number of connections on node 39: 3 
  Number of synaptic contacts on node 39: 26 
  Number of stims on node 39: 11 
Recording 1 traces of 1 types on node 39
Plotting LFP ...
  There was an exception in plotLFP(): 
    module 'netpyne.sim' has no attribute 'allSimData' 
    (<class 'AttributeError'>, AttributeError("module 'netpyne.sim' has no attribute 'allSimData'"), <traceback object at 0x7fd556ea1050>)
Plotting LFP ...
  There was an exception in plotLFP(): 
    module 'netpyne.sim' has no attribute 'allSimData' 
    (<class 'AttributeError'>, AttributeError("module 'netpyne.sim' has no attribute 'allSimData'"), <traceback object at 0x7fd556ea1b40>)

Reading command line arguments using syntax: python file.py [simConfig=filepath] [netParams=filepath]
Loading file ../data/v8_batch4/v8_batch4_0_0_cfg.json ... 
Loading simConfig...
Importing netParams from ../data/v8_batch4/v8_batch4_netParams.py
  Number of cells on node 46: 1 
  Number of connections on node 46: 4 
  Number of synaptic contacts on node 46: 28 
  Number of stims on node 46: 11 
Recording 1 traces of 1 types on node 46
Plotting LFP ...
  There was an exception in plotLFP(): 
    module 'netpyne.sim' has no attribute 'allSimData' 
    (<class 'AttributeError'>, AttributeError("module 'netpyne.sim' has no attribute 'allSimData'"), <traceback object at 0x7f3c3b9e7a50>)
Plotting LFP ...
  There was an exception in plotLFP(): 
    module 'netpyne.sim' has no attribute 'allSimData' 
    (<class 'AttributeError'>, AttributeError("module 'netpyne.sim' has no attribute 'allSimData'"), <traceback object at 0x7f3c3b9e7a50>)

Reading command line arguments using syntax: python file.py [simConfig=filepath] [netParams=filepath]
Loading file ../data/v8_batch4/v8_batch4_0_0_cfg.json ... 
Loading simConfig...
Importing netParams from ../data/v8_batch4/v8_batch4_netParams.py
  Number of cells on node 30: 1 
  Number of connections on node 30: 2 
  Number of synaptic contacts on node 30: 15 
  Number of stims on node 30: 11 
Recording 1 traces of 1 types on node 30
Plotting LFP ...
  There was an exception in plotLFP(): 
    module 'netpyne.sim' has no attribute 'allSimData' 
    (<class 'AttributeError'>, AttributeError("module 'netpyne.sim' has no attribute 'allSimData'"), <traceback object at 0x7ff47fe04cd0>)
Plotting LFP ...
  There was an exception in plotLFP(): 
    module 'netpyne.sim' has no attribute 'allSimData' 
    (<class 'AttributeError'>, AttributeError("module 'netpyne.sim' has no attribute 'allSimData'"), <traceback object at 0x7ff47fe04cd0>)

Reading command line arguments using syntax: python file.py [simConfig=filepath] [netParams=filepath]
Loading file ../data/v8_batch4/v8_batch4_0_0_cfg.json ... 
Loading simConfig...
Importing netParams from ../data/v8_batch4/v8_batch4_netParams.py
  Number of cells on node 47: 1 
  Number of connections on node 47: 9 
  Number of synaptic contacts on node 47: 66 
  Number of stims on node 47: 11 
Recording 1 traces of 1 types on node 47
Plotting LFP ...
  There was an exception in plotLFP(): 
    module 'netpyne.sim' has no attribute 'allSimData' 
    (<class 'AttributeError'>, AttributeError("module 'netpyne.sim' has no attribute 'allSimData'"), <traceback object at 0x7f4493c0b280>)
Plotting LFP ...
  There was an exception in plotLFP(): 
    module 'netpyne.sim' has no attribute 'allSimData' 
    (<class 'AttributeError'>, AttributeError("module 'netpyne.sim' has no attribute 'allSimData'"), <traceback object at 0x7f4493c0b280>)

Reading command line arguments using syntax: python file.py [simConfig=filepath] [netParams=filepath]
Loading file ../data/v8_batch4/v8_batch4_0_0_cfg.json ... 
Loading simConfig...
Importing netParams from ../data/v8_batch4/v8_batch4_netParams.py
  Number of cells on node 25: 1 
  Number of connections on node 25: 3 
  Number of synaptic contacts on node 25: 12 
  Number of stims on node 25: 11 
Recording 1 traces of 1 types on node 25
Plotting LFP ...
  There was an exception in plotLFP(): 
    module 'netpyne.sim' has no attribute 'allSimData' 
    (<class 'AttributeError'>, AttributeError("module 'netpyne.sim' has no attribute 'allSimData'"), <traceback object at 0x7f7b4e2c3870>)
Plotting LFP ...
  There was an exception in plotLFP(): 
    module 'netpyne.sim' has no attribute 'allSimData' 
    (<class 'AttributeError'>, AttributeError("module 'netpyne.sim' has no attribute 'allSimData'"), <traceback object at 0x7f7b4e2c3870>)

Reading command line arguments using syntax: python file.py [simConfig=filepath] [netParams=filepath]
Loading file ../data/v8_batch4/v8_batch4_0_0_cfg.json ... 
Loading simConfig...
Importing netParams from ../data/v8_batch4/v8_batch4_netParams.py
  Number of cells on node 19: 1 
  Number of connections on node 19: 0 
  Number of stims on node 19: 11 
Recording 1 traces of 1 types on node 19
Plotting LFP ...
  There was an exception in plotLFP(): 
    module 'netpyne.sim' has no attribute 'allSimData' 
    (<class 'AttributeError'>, AttributeError("module 'netpyne.sim' has no attribute 'allSimData'"), <traceback object at 0x7f62430a6960>)
Plotting LFP ...
  There was an exception in plotLFP(): 
    module 'netpyne.sim' has no attribute 'allSimData' 
    (<class 'AttributeError'>, AttributeError("module 'netpyne.sim' has no attribute 'allSimData'"), <traceback object at 0x7f62430a6f50>)

Reading command line arguments using syntax: python file.py [simConfig=filepath] [netParams=filepath]
Loading file ../data/v8_batch4/v8_batch4_0_0_cfg.json ... 
Loading simConfig...
Importing netParams from ../data/v8_batch4/v8_batch4_netParams.py
  Number of cells on node 3: 2 
  Number of connections on node 3: 9 
  Number of synaptic contacts on node 3: 61 
  Number of stims on node 3: 22 
Recording 2 traces of 1 types on node 3
Plotting LFP ...
  There was an exception in plotLFP(): 
    module 'netpyne.sim' has no attribute 'allSimData' 
    (<class 'AttributeError'>, AttributeError("module 'netpyne.sim' has no attribute 'allSimData'"), <traceback object at 0x7f70f4091050>)
Plotting LFP ...
  There was an exception in plotLFP(): 
    module 'netpyne.sim' has no attribute 'allSimData' 
    (<class 'AttributeError'>, AttributeError("module 'netpyne.sim' has no attribute 'allSimData'"), <traceback object at 0x7f70f4091050>)

Reading command line arguments using syntax: python file.py [simConfig=filepath] [netParams=filepath]
Loading file ../data/v8_batch4/v8_batch4_0_0_cfg.json ... 
Loading simConfig...
Importing netParams from ../data/v8_batch4/v8_batch4_netParams.py
  Number of cells on node 17: 2 
  Number of connections on node 17: 12 
  Number of synaptic contacts on node 17: 64 
  Number of stims on node 17: 22 
Recording 2 traces of 1 types on node 17
Plotting LFP ...
  There was an exception in plotLFP(): 
    module 'netpyne.sim' has no attribute 'allSimData' 
    (<class 'AttributeError'>, AttributeError("module 'netpyne.sim' has no attribute 'allSimData'"), <traceback object at 0x7f15c79413c0>)
Plotting LFP ...
  There was an exception in plotLFP(): 
    module 'netpyne.sim' has no attribute 'allSimData' 
    (<class 'AttributeError'>, AttributeError("module 'netpyne.sim' has no attribute 'allSimData'"), <traceback object at 0x7f15c79413c0>)

Reading command line arguments using syntax: python file.py [simConfig=filepath] [netParams=filepath]
Loading file ../data/v8_batch4/v8_batch4_0_0_cfg.json ... 
Loading simConfig...
Importing netParams from ../data/v8_batch4/v8_batch4_netParams.py
  Number of cells on node 6: 2 
  Number of connections on node 6: 10 
  Number of synaptic contacts on node 6: 56 
  Number of stims on node 6: 22 
Recording 2 traces of 1 types on node 6
Plotting LFP ...
  There was an exception in plotLFP(): 
    module 'netpyne.sim' has no attribute 'allSimData' 
    (<class 'AttributeError'>, AttributeError("module 'netpyne.sim' has no attribute 'allSimData'"), <traceback object at 0x7f6c72dc3cd0>)
Plotting LFP ...
  There was an exception in plotLFP(): 
    module 'netpyne.sim' has no attribute 'allSimData' 
    (<class 'AttributeError'>, AttributeError("module 'netpyne.sim' has no attribute 'allSimData'"), <traceback object at 0x7f6c72dc3cd0>)

Reading command line arguments using syntax: python file.py [simConfig=filepath] [netParams=filepath]
Loading file ../data/v8_batch4/v8_batch4_0_0_cfg.json ... 
Loading simConfig...
Importing netParams from ../data/v8_batch4/v8_batch4_netParams.py
  Number of cells on node 7: 2 
  Number of connections on node 7: 4 
  Number of synaptic contacts on node 7: 30 
  Number of stims on node 7: 22 
Recording 2 traces of 1 types on node 7
Plotting LFP ...
  There was an exception in plotLFP(): 
    module 'netpyne.sim' has no attribute 'allSimData' 
    (<class 'AttributeError'>, AttributeError("module 'netpyne.sim' has no attribute 'allSimData'"), <traceback object at 0x7f7e0306f6e0>)
Plotting LFP ...
  There was an exception in plotLFP(): 
    module 'netpyne.sim' has no attribute 'allSimData' 
    (<class 'AttributeError'>, AttributeError("module 'netpyne.sim' has no attribute 'allSimData'"), <traceback object at 0x7f7e0306f6e0>)

Reading command line arguments using syntax: python file.py [simConfig=filepath] [netParams=filepath]
Loading file ../data/v8_batch4/v8_batch4_0_0_cfg.json ... 
Loading simConfig...
Importing netParams from ../data/v8_batch4/v8_batch4_netParams.py
  Number of cells on node 42: 1 
  Number of connections on node 42: 3 
  Number of synaptic contacts on node 42: 25 
  Number of stims on node 42: 11 
Recording 1 traces of 1 types on node 42
Plotting LFP ...
  There was an exception in plotLFP(): 
    module 'netpyne.sim' has no attribute 'allSimData' 
    (<class 'AttributeError'>, AttributeError("module 'netpyne.sim' has no attribute 'allSimData'"), <traceback object at 0x7f6cf2582d20>)
Plotting LFP ...
  There was an exception in plotLFP(): 
    module 'netpyne.sim' has no attribute 'allSimData' 
    (<class 'AttributeError'>, AttributeError("module 'netpyne.sim' has no attribute 'allSimData'"), <traceback object at 0x7f6cf2582d20>)

Reading command line arguments using syntax: python file.py [simConfig=filepath] [netParams=filepath]
Loading file ../data/v8_batch4/v8_batch4_0_0_cfg.json ... 
Loading simConfig...
Importing netParams from ../data/v8_batch4/v8_batch4_netParams.py
  Number of cells on node 43: 1 
  Number of connections on node 43: 4 
  Number of synaptic contacts on node 43: 24 
  Number of stims on node 43: 11 
Recording 1 traces of 1 types on node 43
Plotting LFP ...
  There was an exception in plotLFP(): 
    module 'netpyne.sim' has no attribute 'allSimData' 
    (<class 'AttributeError'>, AttributeError("module 'netpyne.sim' has no attribute 'allSimData'"), <traceback object at 0x7f4c4477b9b0>)
Plotting LFP ...
  There was an exception in plotLFP(): 
    module 'netpyne.sim' has no attribute 'allSimData' 
    (<class 'AttributeError'>, AttributeError("module 'netpyne.sim' has no attribute 'allSimData'"), <traceback object at 0x7f4c4477b9b0>)

Reading command line arguments using syntax: python file.py [simConfig=filepath] [netParams=filepath]
Loading file ../data/v8_batch4/v8_batch4_0_0_cfg.json ... 
Loading simConfig...
Importing netParams from ../data/v8_batch4/v8_batch4_netParams.py
  Number of cells on node 14: 2 
  Number of connections on node 14: 10 
  Number of synaptic contacts on node 14: 53 
  Number of stims on node 14: 22 
Recording 2 traces of 1 types on node 14
Plotting LFP ...
  There was an exception in plotLFP(): 
    module 'netpyne.sim' has no attribute 'allSimData' 
    (<class 'AttributeError'>, AttributeError("module 'netpyne.sim' has no attribute 'allSimData'"), <traceback object at 0x7f5988a065a0>)
Plotting LFP ...
  There was an exception in plotLFP(): 
    module 'netpyne.sim' has no attribute 'allSimData' 
    (<class 'AttributeError'>, AttributeError("module 'netpyne.sim' has no attribute 'allSimData'"), <traceback object at 0x7f5988a065a0>)

Reading command line arguments using syntax: python file.py [simConfig=filepath] [netParams=filepath]
Loading file ../data/v8_batch4/v8_batch4_0_0_cfg.json ... 
Loading simConfig...
Importing netParams from ../data/v8_batch4/v8_batch4_netParams.py
  Number of cells on node 15: 2 
  Number of connections on node 15: 13 
  Number of synaptic contacts on node 15: 82 
  Number of stims on node 15: 22 
Recording 2 traces of 1 types on node 15
Plotting LFP ...
  There was an exception in plotLFP(): 
    module 'netpyne.sim' has no attribute 'allSimData' 
    (<class 'AttributeError'>, AttributeError("module 'netpyne.sim' has no attribute 'allSimData'"), <traceback object at 0x7f63bdcbfc30>)
Plotting LFP ...
  There was an exception in plotLFP(): 
    module 'netpyne.sim' has no attribute 'allSimData' 
    (<class 'AttributeError'>, AttributeError("module 'netpyne.sim' has no attribute 'allSimData'"), <traceback object at 0x7f63bdcbfc30>)

Reading command line arguments using syntax: python file.py [simConfig=filepath] [netParams=filepath]
Loading file ../data/v8_batch4/v8_batch4_0_0_cfg.json ... 
Loading simConfig...
Importing netParams from ../data/v8_batch4/v8_batch4_netParams.py
  Number of cells on node 26: 1 
  Number of connections on node 26: 0 
  Number of stims on node 26: 11 
Recording 1 traces of 1 types on node 26
Plotting LFP ...
  There was an exception in plotLFP(): 
    module 'netpyne.sim' has no attribute 'allSimData' 
    (<class 'AttributeError'>, AttributeError("module 'netpyne.sim' has no attribute 'allSimData'"), <traceback object at 0x7fbf8a69bbe0>)
Plotting LFP ...
  There was an exception in plotLFP(): 
    module 'netpyne.sim' has no attribute 'allSimData' 
    (<class 'AttributeError'>, AttributeError("module 'netpyne.sim' has no attribute 'allSimData'"), <traceback object at 0x7fbf8a69bbe0>)

Reading command line arguments using syntax: python file.py [simConfig=filepath] [netParams=filepath]
Loading file ../data/v8_batch4/v8_batch4_0_0_cfg.json ... 
Loading simConfig...
Importing netParams from ../data/v8_batch4/v8_batch4_netParams.py
  Number of cells on node 27: 1 
  Number of connections on node 27: 4 
  Number of synaptic contacts on node 27: 34 
  Number of stims on node 27: 11 
Recording 1 traces of 1 types on node 27
Plotting LFP ...
  There was an exception in plotLFP(): 
    module 'netpyne.sim' has no attribute 'allSimData' 
    (<class 'AttributeError'>, AttributeError("module 'netpyne.sim' has no attribute 'allSimData'"), <traceback object at 0x7f264a95ce60>)
Plotting LFP ...
  There was an exception in plotLFP(): 
    module 'netpyne.sim' has no attribute 'allSimData' 
    (<class 'AttributeError'>, AttributeError("module 'netpyne.sim' has no attribute 'allSimData'"), <traceback object at 0x7f264a95ce60>)

Reading command line arguments using syntax: python file.py [simConfig=filepath] [netParams=filepath]
Loading file ../data/v8_batch4/v8_batch4_0_0_cfg.json ... 
Loading simConfig...
Importing netParams from ../data/v8_batch4/v8_batch4_netParams.py
  Number of cells on node 31: 1 
  Number of connections on node 31: 4 
  Number of synaptic contacts on node 31: 23 
  Number of stims on node 31: 11 
Recording 1 traces of 1 types on node 31
Plotting LFP ...
  There was an exception in plotLFP(): 
    module 'netpyne.sim' has no attribute 'allSimData' 
    (<class 'AttributeError'>, AttributeError("module 'netpyne.sim' has no attribute 'allSimData'"), <traceback object at 0x7f2803844370>)
Plotting LFP ...
  There was an exception in plotLFP(): 
    module 'netpyne.sim' has no attribute 'allSimData' 
    (<class 'AttributeError'>, AttributeError("module 'netpyne.sim' has no attribute 'allSimData'"), <traceback object at 0x7f2803844370>)

Reading command line arguments using syntax: python file.py [simConfig=filepath] [netParams=filepath]
Loading file ../data/v8_batch4/v8_batch4_0_0_cfg.json ... 
Loading simConfig...
Importing netParams from ../data/v8_batch4/v8_batch4_netParams.py

Start time:  2022-03-18 01:06:13.163571

Creating network of 14 cell populations on 48 hosts...
  Number of cells on node 0: 2 
  Done; cell creation time = 3.77 s.
Making connections...
  Number of connections on node 0: 2 
  Number of synaptic contacts on node 0: 14 
  Done; cell connection time = 0.85 s.
Adding stims...
  Number of stims on node 0: 22 
  Done; cell stims creation time = 0.48 s.
Recording 2 traces of 1 types on node 0
0.0s

Running simulation for 300.0 ms...
0.1s
0.2s
0.3s
  Done; run time = 1667.43 s; real-time ratio: 0.00.

Gathering data...
  Done; gather time = 23.07 s.

Analyzing...
  Cells: 66
  Connections: 0 (0.00 per cell)
  Spikes: 714 (36.06 Hz)
  Simulated time: 0.3 s; 48 workers
  Run time: 1667.43 s
   L4_LBC_cAC : 116.667 Hz
   L4_LBC_cNA : 116.667 Hz
   L4_LBC_cST : 110.000 Hz
   L4_LBC_dNA : 130.000 Hz
   L4_LBC_dST : 113.333 Hz
   L4_PC_cAD : 25.802 Hz
   L5_MC_bAC : 40.000 Hz
   L5_MC_bIR : 43.333 Hz
   L5_MC_bST : 20.000 Hz
   L5_MC_cAC : 115.000 Hz
   L5_MC_cNA : 106.667 Hz
   L5_MC_cST : 96.667 Hz
   L5_MC_dNA : 103.333 Hz
   L5_TTPC1_cAD : 16.667 Hz
Saving output as ../data/v8_batch4/v8_batch4_0_0.pkl ... 
Finished saving!
Saving output as ../data/v8_batch4/v8_batch4_0_0.json  ... 
Finished saving!
  Done; saving time = 18.32 s.
Plotting 2D representation of network cell locations and connections...
Plotting LFP ...
Plotting 3D cell shape ...
Plotting raster...
Plotting recorded cell traces ... cell
  Done; plotting time = 18.62 s

Total time = 1740.55 s

End time:  2022-03-18 01:35:13.714142
Plotting LFP ...
Plotting LFP ...
[mpiexec@fernando-Nitro5] Sending Ctrl-C to processes as requested
[mpiexec@fernando-Nitro5] Press Ctrl-C again to force abort
